samples/python/agents/pydantic_agent/__main__.py

```diff
@@ -0,0 +1,9 @@
+from a2a.server import A2AServer
```

Comment:
Where is the import `a2a` coming from? I only see `common` and `server` being exported [per pyproject.toml](https://github.com/google/A2A/blob/cd9eb5e66219e4afab4e35985b2514bda5943b1f/samples/python/pyproject.toml)